### DISCUSSION TITLE
Some idéas trowed like this...

### DISCUSSION THREAD

**Discussion by Raph99-Thanys** [neumatho/NostalgicPlayer#51] [Ideas]
Hi, me again ^^'

I trow here some ideas for NostalgicPlayer (or not lol ) , it's just some ideas and it's up to you ;)

- Keyboard Multimedia Key Support.
- Auto Hidding unused file format in the songlist windows (like the .smpl in TMFX files system who are not used and make error message if used (yes, we can remove them ^^' with the [-] button, just something more intuitive for newbie ;) )
- Darkmode UI
- Cross or something for closing the error message ^^" Alt+F4 not really intuitive in this case.
- In the playlist, the possibility of saving which "subsong" we want to play (as for example on the song: "Slam Tilt - Demon Song" where I would like "NostalicPlayer" to directly read subsong 6 of 13

Ty for read. NostaligPlayer was already great i love it, and more than all i USE it! ty again for this :)

**Comment by neumatho** [comment]
Thanks for the ideas. I will write them on a list and see which one I will implement. I will not promise when they eventually will be implemented.

**Comment by neumatho** [comment]
> * In the playlist, the possibility of saving which "subsong" we want to play (as for example on the song: "Slam Tilt - Demon Song" where I would like "NostalicPlayer" to directly read subsong 6 of 13

This has been implemented and will be released in version 2.4.0.

**Reply by Raph99-Thanys** [reply]
Nice! ty :)
